InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -183,6 +199,7 @@ public void periodic(float deltaTime) {
                 targetPositionElevation_ticks = (int) -backwardElevationSoftLimit_ticks;
             }
         }
+        // Checks if the limit switch for the elevation motor is closed, if it is: set the motor's sensor position to ELEVATION_LIMIT_SWITCH_DEG.

```

Comment:
It would be good to explain why we set it to ELEVATION_LIMIT_SWITCH_DEG and not 0.

This is because the limit switch might trigger a tiny bit before the elevation motor reaches it's zero point, so setting it to 1.1 degrees (or something like that) would mean out actual 0 is when the elevation motor is completely down.